### DISCUSSION TITLE
Zoraxy on Windows: Where are the logs located?

### DISCUSSION THREAD

**Discussion by AmIBeingObtuse** [tobychui/zoraxy#188] [Q&A]
Hi

Absolutely love this project. I can finaly found a great reverse proxy to run on windows outside of docker onus to the windows binary and get the real ip of clients connecting.

My question is where are the logs located? I was thinking of connecting up something like Grafana or other aggregating log tools to view connections and access attempts ect...

On checking the Zoraxy folder I cannot see any access logs. More than likely just me being blind.

Also checked through the other discussion's and issues couldn't find anything similar. 

Thanks.

**Comment by tobychui** [comment]
Currently there are no log file implemented in Zoraxy for storing raw HTTP requests (only system info and errors are logged in current implementation). 

The incoming HTTP requests are analysis in real-time and only the data required for displaying in statistical analysis (e.g. client Country code, ip address) are stored and the rest is discarded. If you need such feature, feel free to open a feature requests and describe what you want in details.

**Comment by AmIBeingObtuse** [comment]
Hi

Okay thanks I will definitely do that.

That's a big must for a reverse proxy so I appreciate your quick reply.
